api/kyverno/constants.go

```diff
@@ -7,6 +7,7 @@ const (
 	LabelCacheEnabled     = "cache.kyverno.io/enabled"
 	LabelCertManagedBy    = "cert.kyverno.io/managed-by"
 	LabelCleanupTtl       = "cleanup.kyverno.io/ttl"
+	AnnotationCleanupPropagationPolicy = "cleanup.kyverno.io/propagation-policy"
```

Comment:
Can you put that in the annotations section ? (a couple of lines below)

Reply:
Okay